app/mailers/subscription_mailer.rb

```diff
@@ -23,4 +23,14 @@ def notify_note_liked(node, user)
     @footer = feature('email-footer')
     mail(to: node.author.email, subject: subject).deliver
   end
+
+  def notify_tag_added(node,tag)
+    given_tags = node.tags
+    users_to_email = tag.followers_who_dont_follow_tags(given_tags)
```

Comment:
That that does not work `NoMethodError: undefined method tag'for #<Tag:0x00000009a457a0>`
for that specif test. On running all the tests the error is `ActiveRecord::RecordNotFound: Couldn't find User without an ID`